lib/cli.js

```diff
@@ -78,10 +78,23 @@ function init() {
     // start the interactive UI if no generator is passed
     if (!cmd) {
       if (opts.help) {
```

Comment:
All this block should go into a function. e.g.:

```js
var generatorList = createGeneratorList(env.getGeneratorsMeta());
console.log(generatorList);
```

This would also allow us to add unit tests validating the fix is working correctly.